phpunit/blocks/renderBlockCorePostExcerpt.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Post Excerpt rendering tests.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ * @covers ::gutenberg_render_block_core_post_excerpt
+ * @group blocks
+ */
```

Comment:
I've just applied some formatting.

```suggestion
/**
 * Tests for the gutenberg_render_block_core_post_excerpt() function.
 *
 * @package WordPress
 * @subpackage Blocks
 *
 * @covers ::gutenberg_render_block_core_post_excerpt
 * @group blocks
 */
```